test/functional/test-performance.js

```diff
@@ -14,7 +14,11 @@
  * limitations under the License.
  */
 
-import {installPerformanceService} from '../../src/service/performance-impl';
+import * as sinon from 'sinon';
```

Comment:
That's fine, sinon is globally installed in the test framework. We actually have a pending issue to remove all these imports.